training/general/src/environments/acpt-pytorch-1.11-py38-cuda11.5-gpu/environment.yaml

```diff
@@ -0,0 +1,33 @@
+image:
+  name: azureml/curated/acpt-pytorch-1.11-py38-cuda11.5-gpu
```

Comment:
The pre-registry version of this environment [here](https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/envs/Latest/AzureML-PTA-pytorch-1.11-py38-cuda11.5-gpu/.contact) will need to be turned off before this one is released.

Reply:
Do I remove the publish section from .contact file in Azuremlcli repo?

Reply:
checked in the change to turn off the pre regitry version. Will move to deprecate folder later.